pre_commit_linter.py

```diff
@@ -83,15 +83,15 @@
 _MESSAGE_TYPE_SUCCESS = 'SUCCESS'
 _MESSAGE_TYPE_FAILED = 'FAILED'
 
-EXCLUDED_PATHS = ('third_party/*',)
+EXCLUDED_PATHS = ('third_party/*', '.git/*', '.github/*')
 
 PROTOTOOL_CONFIG_FILE = 'prototool_config.json'
 
 def _get_changed_filenames():
-    """Returns a list of modified files (both staged and unstaged)
+    """Returns a list of modified files (both staged and unstaged).
 
     Returns:
-        a list of filenames of modified files.
+        List(str). A list of filenames of modified files.
```

Comment:
list(str).

Ditto elsewhere -- types should not be capitalized. Just the description after that.

Reply:
Done in #2.